did a few changes to make it look better

theiconik opened the pull request.

theiconik committed:
did a few changes to make it look better

ancoreraj commented:
Thank you  @theiconik , I will review and merge it tomorrow.
Thanks for the contribution.

ancoreraj commented:
@theiconik Can you share some SS?

theiconik commented:
**1. Gave a little padding from the bottom of nav to the nav items and made the logo transparent**
![Screenshot (1)](https://user-images.githubusercontent.com/52995399/145220958-fa144090-5dd2-45f4-a562-1fdd0cf7599a.png)

 **2. Thought a lot about motive cards, making the center one bigger than side ones was a good idea but on a smaller window, it looked weird, so changed it to all equal sizes and added icons or say emojis!**
![Screenshot (2)](https://user-images.githubusercontent.com/52995399/145220977-6c5e9502-df7b-44c2-8e70-ec392cbe6f2f.png)

 **3. Made discord logo transparent and rotate animation on hover**
![Screenshot (3)](https://user-images.githubusercontent.com/52995399/145220985-79a124c7-6e87-4436-9a3d-67b1a07fa8fd.png)

 **4. Removed header image from about us coz trust me it was looking really bad and did something. Look for yourself!**
![Screenshot (4)](https://user-images.githubusercontent.com/52995399/145220992-ac1c0be9-14d5-41c6-bd81-855161e08c45.png)

 **5. Removed borders from badges and changed the team badges background to blue ig.**
![Screenshot (5)](https://user-images.githubusercontent.com/52995399/145220997-b2722bdc-d9ed-42cd-8689-c384f7b23954.png)
